xml/System.Globalization/ISOWeek.xml

```diff
@@ -74,10 +84,10 @@
         <Parameter Name="year" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="year">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="year">Represents an ISO week-numbering year (also called ISO year informally).</param>
+        <summary>calculate the number of weeks in a given ISO week-numbering year.</summary>
+        <returns>The number of ISO weeks in the year. Returns either 52 or 53.</returns>
+        <remarks></remarks>
```

Comment:
If you don't plan on adding remarks for a member, you can leave the default value "To be added.".

I don't know if removing the default string would mess the build process or the statistics.

@mairaw @rpetrusha does this matter to you?

Reply:
From the viewpoint of the build, empty remarks don't pose a problem. But we regularly search for "To be added." to see where we're missing documentation, so adding back "To be added." can be helpful.

Reply:
```suggestion
        <remarks>To be added.</remarks>
```

Reply:
@rpetrusha but this is not missing documentation. We just need to have this section empty. could you please clarify?

Reply:
Yes. The string "To be added." is automatically removed from the build, so it doesn't appear in the documentation. We can search for the string to easily identify sections that we haven't documented, so it's useful to leave it in the documentation rather than removing it.